src/index.html

```diff
@@ -2,16 +2,254 @@
+            class="input"
+          >
+        </label>
+        <label 
+          for="birthday" 
+          class="label"
+        >
+          Full date of birth:
+          <input 
+            type="date" 
+            id="birthday" 
+            name="birthday" 
+            required
+            class="input"
+          >
+        </label>
+        <label 
+          for="agreement" 
+          class="label"
+        >
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            id="agreement" 
+            name="agrrement" 
+            required
+            class="input"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset second">
+        <legend>Registration</legend>
+        <label
+          for="email" 
+          class="label"
+        >
+          E-mail:
+          <input 
+            type="email" 
+            id="email" 
+            name="email" 
+            placeholder="email@example.com"
+            class="input"
+          >
+        </label>
+        <label 
+          for="password"
+          class="label"
+        >
+          Password:
+          <input 
+            type="password" 
+            id="password" 
+            name="password"
+            class="input"
+          > 
+        </label>
+      </fieldset>
+      <fieldset class="fieldset third">
+        <legend>An interesting fact about you!</legend>
+        <label 
+          for="cats" 
+        >
+          Do you love cats?
+          <input 
+            type="radio" 
+            id="cats" 
+            name="cats"
+            class="input"
+          >
+          <span>yes</span>
+          <input 
+            type="radio" 
+            id="cat" 
+            name="cat"
+            class="input"
+          >
+          <span>No</span>
+        </label>
+        <label 
+          for="color"
+          class="label"
+        >
+          What is your favorite color?
+          <input 
+            type="color" 
+            id="color" 
+            name="color" class="input"
+          >
+        </label>
+        <label 
+          for="time"
+          class="label"
+        >
+          What time do you go to bed?
+          <input 
+            type="time" 
+            id="time" 
+            name="time" 
+            step="1"
+            class="input"
+          >
+        </label>
+        <label 
+          for="cars"
+          class="label"
+        >
+          What are your favorite brand of cars?
+          <select 
+            name="cars" 
+            id="cars" 
+            multiple
+            class="input"
+          >
+            <option 
+              value="Bmw"
+            >
+              BMW
+            </option>
+            <option 
+              value="Audi"
+            >
+              Audi
+            </option>
+            <option 
+              value="lada"
+            >
+              Lada
+            </option>
```

Comment:
in this particular case you've kinda over-indented attributes ))
you need to move attributes to new lines when there are 3 and more attributes.
2 attributes are totally fine to stick at one line.

another context - better pass lower-case values in attribute `value` - it'll mitigate some issues when working with data at back-end
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
```